src/commands/account/add_key/autogenerate_new_keypair/save_keypair_to_keychain/mod.rs

```diff
@@ -12,18 +12,32 @@ impl SaveKeypairToKeychain {
         config: crate::config::Config,
         key_pair_properties: crate::common::KeyPairProperties,
         prepopulated_unsigned_transaction: near_primitives::transaction::Transaction,
+        is_save_to_macos_keychain: bool,
```

Comment:
It is better to use an enum as we potentially will add more keychain options, and `bool` is not going to be enough.

Reply:
removed "is_save_to_macos_keychain"